model/src/main/kotlin/config/LicenseChoices.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.spdx.SpdxExpression
+
+/**
+ * The license choices configured for a repository.
+ */
+data class LicenseChoices(
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val packageLicenseChoices: List<PackageLicenseChoice> = emptyList()
+)
+
+/**
+ * [LicenseChoice]s defined for an artifact.
+ */
+data class PackageLicenseChoice(
+    val packageId: Identifier,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val licenseChoices: List<LicenseChoice> = emptyList()
+)
+
+/**
+ * An individual license choice.
+ *
+ * Given is the complete license expression the license choice is going to be applied on.
+ * When a sub-expression is supplied the choice will be applied only to the sub-expression should it be contained in
```

Comment:
I think this should be "If a sub-expression is supplied" instead of "When", analogously to the second sentence. Don't miss the comma between if and main clause.